### DISCUSSION TITLE
Random Tips on loading screen

### DISCUSSION THREAD

**Discussion by Flactine** [Phobos-developers/Phobos#1016] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

Display random tips for players who don't yet know the mechanics of game (or mod).

### INI code

```ini
;in Ra2md.ini
[Phobos]
EnableRandomTips=no	; boolean

;in rulesmd.ini
[Tips]
0=SomeTip
...

[SomeTip]
UIName=Name:SomeTip  ;Tips text in CSF
Name=SomeTip
TipOwner= (list of HouseTypes)  ;Only these side players can have chance to see this tip when loading
DisplayWhenLoading=no     ;boolean
```

**Comment by Flactine** [comment]
Example:
;in Ra2md.ini
[Phobos]
EnableRandomTips=no	; boolean

;in rulesmd.ini
[Tips]
0=AlliedTip1
1=FranceTip1
2=SovietTip1

[AlliedTip1]
UIName=Name:AlliedTip1  ;Tips text in CSF
Name=AlliedTip1
TipOwner=British,French,Germans,Americans,Alliance ;Only these players can have chance to see this tip when loading
DisplayWhenLoading=no     ;boolean

[FranceTip1]
UIName=Name:FranceTip1  ;Tips text in CSF
Name=FranceTip1
TipOwner=French           ;Only these players can have chance to see this tip when loading
DisplayWhenLoading=no     ;boolean

[SovietTip1]
UIName=Name:SovietTip1    ;Tips text in CSF
Name=SovietTip1
TipOwner=Russians,Confederation,Africans,Arabs ;Only these players can have chance to see this tip when loading
DisplayWhenLoading=no      ;boolean

**Comment by 8abs** [comment]
This one sounds wonderful.
